### PR TITLE
Add link to material items in MainUI

### PR DESCRIPTION
**Pull request by eikowagenknecht**
The material items (see #604) can be used in various places in the UI, but it's not documented (yet). Yannick himself also [seems to use this](https://community.openhab.org/t/wiki-building-pages-in-the-oh3-ui-documentation-draft-2-3/104392/117).

This PR adds a link in the UI to the existing two (OH icons, f7 icons, **Material icons**).

Before:
image

After:
image

Signed-off-by: Eiko Wagenknecht <eiko.wagenknecht@web.de>

### CODE REVIEW

File: bundles/org.openhab.ui/web/src/assets/definitions/widgets/standard/cards.js [hunk 1]
```diff
@@ -24,8 +24,7 @@ export const OhLabelCardDefinition = () => new WidgetDefinition('oh-label-card',
     pt('background', 'Background style', 'Background style (in CSS "background" attribute format)'),
     pt('fontSize', 'Font Size', 'Font size (e.g. "34px")'),
     pt('fontWeight', 'Font Weight', 'Font weight (e.g. "normal" or "bold")'),
-    pt('icon', 'Icon', 'Use <code>oh:iconName</code> (<a class="external text-color-blue" target="_blank" href="https://www.openhab.org/link/icons">openHAB icon</a>) or <code>f7:iconName</code> (<a class="external text-color-blue" target="_blank" href="https://framework7.io/icons/">Framework7 icon</a>)'),
-    pt('iconColor', 'Icon Color', 'Not applicable to openHAB icons').a(),
+    pt('icon', 'Icon', 'Use <code>oh:iconName</code> (<a class="external text-color-blue" target="_blank" href="https://www.openhab.org/link/icons">openHAB icon</a>), <code>f7:iconName</code> (<a class="external text-color-blue" target="_blank" href="https://framework7.io/icons/">Framework7 icon</a>) or <code>material:iconName</code> (<a class="external text-color-blue" target="_blank" href="https://jossef.github.io/material-design-icons-iconfont/">Material icon</a>)'),    pt('iconColor', 'Icon Color', 'Not applicable to openHAB icons').a(),
```
Comment (ghys) [inline bundles/org.openhab.ui/web/src/assets/definitions/widgets/standard/cards.js:27]: Please add the line break back.

**Review by ghys — CHANGES_REQUESTED** [review]
Thanks!

File: bundles/org.openhab.ui/web/src/assets/definitions/widgets/standard/cards.js (hunk 1 above)
Comment (eikowagenknecht) [reply bundles/org.openhab.ui/web/src/assets/definitions/widgets/standard/cards.js:27]: Huh, don‘t even know how it disappeared. Will add it back of course.

**Comment by eikowagenknecht** [comment]
Fixed the linebreak issue.

**Review by ghys — APPROVED** [review]

**Pull request merged**